odbc-api/src/handles/connection.rs

```diff
@@ -186,4 +186,25 @@ impl<'c> Connection<'c> {
                 .into_result(self)
         }
     }
+
+    /// Get the name of the database management system used by the connection.
+    pub fn database_management_system_name(
+        &self,
+        name: &mut OutputStringBuffer,
+    ) -> Result<(), Error> {
+        let out_name = name.mut_buf_ptr();
+        let out_buf_len = name.buf_len();
```

Comment:
According to the documentation, this is supposed to be the length in bytes, not characters. Manual tests with short buffer lengths can help figuring out the truth.

Reply:
Yeah that's correct, thanks for catching this!